src/RangeWithValue/index.tsx

```diff
@@ -31,6 +32,7 @@ export function RangeWithValue({
   actualValue,
   rangeType,
   bufferPercentage = 0.1,
+  showMean = false,
```

Comment:
A bit shorter, Typescript will interpret `undefined` as falsy below.

```suggestion
  showMean,
```